packages/medusa-fulfillment-webshipper/src/services/webshipper-fulfillment.js

```diff
@@ -9,6 +9,16 @@ class WebshipperFulfillmentService extends FulfillmentService {
 
     this.options_ = options
 
+    if (!options.coo_countries) {
```

Comment:
Add check for empty array in case someone has `coo_countries: []`

Reply:
I think this is fair use if you don't want CoO to be sent at all

Reply:
Now, that I come to think of it, I don't think that CoO should be attached by default. It should be an active choice since I would assume that CoO's are more often omitted than included. But correct me, if I am wrong.

Reply:
```js
        const cooCountries = this.options_.coo_countries
        if (
          (cooCountries.includes("all") ||
            cooCountries.includes(
              shipping_address.country_code.toLowerCase()
            )) &&
          this.invoiceGenerator_.createCertificateOfOrigin
        ) {
```
The way it behaves is that if you have a certificate of origin generator attached it will generate for all countries. So the option only takes effect when you have the generator

Reply:
Alright makes sense. So one is making an active choice. I'll approce